examples/dwarfdump.rs

```diff
@@ -161,15 +161,18 @@ fn add_relocations(
             object::RelocationKind::Absolute => {
                 match relocation.target() {
                     object::RelocationTarget::Symbol(symbol_idx) => {
-                        if let Some(symbol) = file.symbol_by_index(symbol_idx) {
-                            let addend = symbol.address().wrapping_add(relocation.addend() as u64);
-                            relocation.set_addend(addend as i64);
-                        } else {
-                            println!(
-                                "Relocation with invalid symbol for section {} at offset 0x{:08x}",
-                                section.name().unwrap(),
-                                offset
-                            );
+                        match file.symbol_by_index(symbol_idx) {
+                          Ok(symbol) => {
```

Comment:
This uses two spaces as indent instead of four like the rest of this file.

Reply:
Ah, right. Fixed it.